cob_bringup/components/canopen_base.launch

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0"?>
 <launch>
 	<arg name="robot"/>
-	<arg name="component_name" value="base"/>
+	<arg name="component_name" default="base"/>
```

Comment:
I guess `value` vs. `default` was intended by @ipa-mdl, as it is a base-specific launch file...

Don't know about the parameters below